test/llvm_test_code/xtaint/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# FIXME
+set(XTAINT_SOURCES
+    xtaint01.cpp
+    # xtaint01_json.cpp
```

Comment:
The `xtaint01_json.cpp` source is commented out, because it is now separately compiled with debug information (that are now needed by the new `TaintConfig`).

The other both sources are out-commented, because with the new configuration there is -as explained in a previous comment- now way to specify the same sinks. However, since the old configuration is already removed, these tests do not compile any more.